manifests/s/schild-report/schild-report/2.5.0.0/schild-report.schild-report.locale.en-US.yaml

```diff
@@ -7,7 +7,6 @@ PackageLocale: en-US
 Publisher: ${env.THE_AUTHOR}
```

Comment:
Is that really what the publisher's name is?

Reply:
Yup 🤦‍♂️

![image](https://user-images.githubusercontent.com/52826317/131561944-effd5c57-64f2-44ca-aef6-011dfaa9107e.png)

Reply:
Yes, Yes it is 😬
If you can help him he would be very happy 😁 https://github.com/schild-report/schild.report/issues/6